x-pack/plugins/enterprise_search/server/routes/enterprise_search/search_applications.ts

```diff
@@ -46,6 +47,12 @@ export function registerSearchApplicationsRoutes({ log, router }: RouteDependenc
         request.query
       )) as EnterpriseSearchApplicationsResponse;
 
+      await Promise.all(
```

Comment:
I think we want some error handling here, to nicely display some information about a search application. This is what I see when I try to view search applications after manually removing the alias: 

<img width="1025" alt="image" src="https://github.com/elastic/kibana/assets/63422879/4e6457ef-77e0-496b-914a-7c7c7e2caf71">